lib/platform/gitlab/index.js

```diff
@@ -264,6 +264,9 @@ async function commitFilesToBranch(
   if (pr) {
     logger.debug('Reopening PR');
     await reopenPr(pr.number);
+    // Close and repoen MR again due to known GitLab bug https://gitlab.com/gitlab-org/gitlab-ce/issues/41545
+    await closePr(pr.number);
+    await reopenPr(pr.number);
```

Comment:
Do we need two reopenPr() in this function?

Reply:
I think we do. The first reopenPr() is to reopen the MR as indicated in #1657. And the second is to reopen the MR the close MR above it.